static/app/views/explore/tables/columnEditorModal.spec.tsx

```diff
@@ -182,7 +193,12 @@ describe('ColumnEditorModal', () => {
       ['span.duration', 'number'],
       ['span.op', 'string'],
     ];
-    await userEvent.click(screen.getByRole('button', {name: 'Column project string'}));
+
+    const projectColumn = screen.getAllByTestId('editor-column')[1]!;
+
+    await userEvent.click(
+      within(projectColumn).getByRole('button', {name: 'Column project string'})
+    );
```

Comment:
tbh, I’m not really sure why I had to change all these tests. I got a message that buttons with that name exist more than once, and from what I could tell, the whole row is a button with the same accessibility label. But I’m not sure what my PR changed to surface this or how this could’ve worked on master.

My best guess is that it worked coincidentally, I’m now just narrowing the scope of the selector to make the tests pass again. It’s hard to investigate if you don’t get to physically “see” what gets rendered though...